First attemp to see pics when sending images

aarizkuren opened the pull request:
Con este PR se guarda la imagen enviada en el chat, pero hay que salir de el y volver a entrar para poder verlo

issue #168

aarizkuren committed:
First attemp to see pics when sending on whatsapp.

aarizkuren committed:
Merge remote-tracking branch 'upstream/master'

Gioyik commented:
Quieres decir que para ver la imagen en la conversación hay que cerrar y abrir Loqui de nuevo?

aarizkuren commented:
No, con volver al listado de chats y volver a entrar en esa conversacion vale. Lo guardo en el object chats pero no lo pinto todavia por pantalla, y al volver a entrar en la pantalla, como vuelve a pintar todo el object chat, pues ya se ve. Cuando tenga un rato, intento hacer que se vea, o si alguien ve como hacerlo, adelante =)

aarizkuren committed:
#179 refactored code
  Notification method update on file uploading

aesedepece commented:
El mensaje no se está mostrando por defecto porque no se está enviando mediante `Message.send()`. Tenemos varias opciones:
- Hacer que los mensajes multimedia se envíen también mediante `Message.send()`.
- Añadir el mensaje gráficamente desde el `addMediaMessageToChat` con el siguiente código:

``` javascript
$('section#chat ul#messages li:last-child').append(msg.preRender());
```

aarizkuren commented:
Sí eso ya lo tengo programado, lo unico tengo que hacer aún alguna prueba más, porque no he guardado en id del mensaje en el object Message y al hacer el preRender lanza un exception, en cuanto solucione esto, subo los cambios y os aviso, pero creo que en un rato lo tendre

aarizkuren committed:
#179 refactored code
  Notification method update on file uploading
  Added uuid function on tools to patch id generation on file sending
  sended image show

aarizkuren committed:
Merge remote-tracking branch 'upstream/master'

aarizkuren commented:
Listo, ahora cuando mandamos una imagen, se muestra en el chat. Falta añadir que ocurra lo mismo para el video y el audio.

No he conseguido coger el id que deberia traer de vuelta el mensaje, creo que tendriamos que modificar coseme para que nos lo devolviese, pero por ahora, he creado una funcion en Tools que crea un uuid aleatorio, y se lo asigno como id, para que sea dificil que los ids coincidan.

Probarlo, y si lo veis bien, haced el merge :)

aarizkuren committed:
#186 Confirm Location recieve support
  fixed location received support

aarizkuren commented:
Fallo, no queria subir este cambio a este PR, pero bueno ya que esta... lo explico en #186

aarizkuren committed:
Merge remote-tracking branch 'upstream/master'

aarizkuren committed:
#186 Confirm Location recieve support
  added gmaps open url to location img

aarizkuren committed:
#186 Confirm Location recieve support
  refactored code
  added css to maps

aarizkuren commented:
Buff, @aesedepece, @Gioyik podeis echarle un ojo a este PR
como lo iba haciendo en master, no puedo crear ramas para cada issue hasta que este PR este merged, bueno se puede pero es un poco jaleo. 

He añadido unos cambios de css y alguna refactorización en la acción para recibir la localización. Ahora funciona casi bien, en #186 señalo que es lo que falla.

aarizkuren committed:
Merge remote-tracking branch 'upstream/master'

Gioyik commented:
Por mi +1 al merge, y lo miramos que falta en #186.
